### PR TITLE
Julia printer

### PR DESCRIPTION
**Pull request by isuruf**
cc @ChrisRackauckas

### CODE REVIEW

**Comment by ChrisRackauckas** [comment]
Thanks!

File: symengine/printer.h [hunk 1]
```diff
@@ -250,6 +250,13 @@ class StrPrinter : public BaseVisitor<StrPrinter>
     std::string apply(const vec_basic &v);
     std::string apply(const Basic &b);
 };
+
+class JuliaStrPrinter : public BaseVisitor<JuliaStrPrinter, StrPrinter>
+{
+    virtual void _print_pow(std::ostringstream &o, const RCP<const Basic> &a,
+                            const RCP<const Basic> &b);
+    void bvisit(const Constant &x);
+}
```
Comment (ranjithkumar007) [inline symengine/printer.h]: `;` missing here.

Comment (isuruf) [inline symengine/printer.h]: @ranjithkumar007, can you review?

**Comment by isuruf** [comment]
Thanks for the reviews

**Pull request merged**